app/tests/e2e/test_issuer.py

```diff
@@ -89,35 +97,117 @@ async def test_get_records(alice_member_client: AsyncClient):
-    result = response.json()
-
-    assert response.status_code == 400
-    assert_that(result).contains("detail")
-    assert "in offer_sent state (must be offer_received)" in result["detail"]
+    response = await alice_member_client.get(
+        BASE_PATH,
+        params={"connection_id": bob_and_alice_connection["alice_connection_id"]},
+    )
+    assert check_webhook_state(
+        client=alice_member_client,
+        desired_state={"state": "offer-received"},
+        topic="issue_credential",
+    )
 
 
 @pytest.mark.asyncio
 async def test_store_credential(
-    bob_member_client: AsyncClient, credential_exchange_id: str
+    alice_member_client: AsyncClient,
+    bob_member_client: AsyncClient,
+    credential_exchange_id: str,
+    bob_and_alice_connection: BobAliceConnect,
+    schema_definition: SchemaSendResult,
 ):
-    # TODO check for the correct response when state is credential_received
-    # We can't complete this with auto accept enabled
-    time.sleep(5)
+    credential = {
+        "protocol_version": "v1",
+        "connection_id": bob_and_alice_connection["bob_connection_id"],
+        "schema_id": schema_definition.schema_id,
+        "attributes": {"speed": "average"},
+    }
+
+    await register_issuer(bob_member_client, schema_definition.schema_id)
+
     response = await bob_member_client.post(
-        f"{BASE_PATH}/{credential_exchange_id}/store"
+        BASE_PATH,
+        json=credential,
+    )
+    credential_exchange = response.json()
+    assert credential_exchange["protocol_version"] == "v1"
+
+    assert check_webhook_state(
+        client=bob_member_client,
+        filter_map=FilterMap(
+            filter_key="credential_id",
+            filter_value=credential_exchange["credential_id"],
+        ),
+        desired_state={"state": "offer-sent"},
+        topic="issue_credential",
+    )
+
+    response = await alice_member_client.get(
+        BASE_PATH,
+        params={"connection_id": bob_and_alice_connection["alice_connection_id"]},
+    )
+    assert check_webhook_state(
+        client=alice_member_client,
+        desired_state={"state": "offer-received"},
+        topic="issue_credential",
+    )
+
+    cred_hooks = get_hooks_per_topic_per_wallet(
+        client=alice_member_client, topic="issue_credential"
+    )
+
+    cred_hook = [h for h in cred_hooks if h["payload"]["state"] == "offer-received"][0]
+    credential_id = cred_hook["payload"]["credential_id"]
+
+    response = await alice_member_client.post(f"{BASE_PATH}/{credential_id}/request")
+
+    assert check_webhook_state(
+        client=bob_member_client,
+        desired_state={"state": "request-received"},
+        topic="issue_credential",
+    )
+
+    response = await alice_member_client.post(f"{BASE_PATH}/{credential_id}/store")
+
+    assert check_webhook_state(
+        client=alice_member_client,
+        desired_state={"state": "credential-received"},
+        topic="issue_credential",
     )
```

Comment:
Agreed on the latter bit. Changed that now. 

For the first paragraph:

* Bob sends a credential (offer) (automating the entire flow)
* Alice receives the offer
* as per [diagram](https://github.com/hyperledger/aries-rfcs/blob/main/features/0036-issue-credential/credential-issuance.png) alice then sends a credential request (data is correct)
* Bob replies with the credential (because we have used 'automating the entire flow - `app/generic/issuer/issuer.py::send_credential`)

=> that's how I understood this to work. Maybe I'm making a mistake here?

added a check for `credential-acked` to the test and moved the `credential-received` check in front of storing it.